pkg/http/http.go

```diff
@@ -259,6 +260,8 @@ func (c *Client) Discover(ctx context.Context) {
 }
 
 // Resolve refreshes and resolves the list of targets.
-func (c *Client) Resolve(ctx context.Context) {
-	c.provider.Resolve(ctx, append(c.fileSDCache.Addresses(), c.staticAddresses...))
+func (c *Client) Resolve(ctx context.Context) tsdberrors.MultiError {
+	// Skip the error as it would be logged in the corresponding component.
```

Comment:
So why we return error then?

Reply:
Also what do you mean skip? We are not skipping , we are passing it further so handling properly

Reply:
I have changed it.